doc/source/geotransformations.md

```diff
@@ -0,0 +1,204 @@
+{func}`~geoutils.Raster.reproject`, {func}`~geoutils.Raster.crop` and {func}`~geoutils.Raster.shift`. Additionally, 
+for convenience and consistency during analysis, most operations can be passed a {class}`~geoutils.Raster` or 
+{class}`~geoutils.Vector` as a reference to match.
+In that case, no other argument is necessary. For more details, see {ref}`core-match-ref`.
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+pyplot.rcParams['font.size'] = 9
+```
+
+## Reproject
+
+{func}`geoutils.Raster.reproject` or {func}`geoutils.Vector.reproject`.
+
+Reprojections **transform geospatial data from one CRS to another**.
+
+For vectors, the transformation of geometry points is exact. However, in the case of rasters, the projected points 
+do not necessarily fall on a regular grid and require re-gridding by a 2D resampling algorithm, which results in a slight 
+loss of information (value interpolation, propagation of nodata).
+
+For rasters, it can be useful to use {func}`~geoutils.Raster.reproject` in the same CRS simply for re-gridding, 
+for instance when downsampling to a new resolution {attr}`~geoutils.Raster.res`.
+
+```{tip}
+Due to the loss information when re-gridding, it is important to **minimize the number of reprojections during the 
+analysis of rasters** (performing only one, if possible). For the same reason, when comparing vectors and rasters in 
+different CRSs, it is usually **better to reproject the vector with no loss of information, which is the default 
+behaviour of GeoUtils in raster–vector–point interfacing**.
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import matplotlib.pyplot as plt
+import geoutils as gu
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+rast.set_nodata(0)  # Annoying to have to do this here, should we update it in the example?
+rast2 = gu.Raster(gu.examples.get_path("everest_landsat_b4_cropped"))
+vect = gu.Vector(gu.examples.get_path("everest_rgi_outlines"))
+```
+
+```{code-cell} ipython3
+# Reproject vector to CRS of raster by simply passing the raster
+vect_reproj = vect.reproject(rast)
+# Reproject raster to smaller bounds and different X/Y resolution
+rast_reproj = rast.reproject(
+    res=(rast.res[0] * 2, rast.res[1] / 2),
+    bounds={"left": rast.bounds.left, "bottom": rast.bounds.bottom, 
+            "right": rast.bounds.left + 10000, "top": rast.bounds.bottom + 10000},
+    resampling="cubic")
+```
+
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show the code for plotting the figure"
+:  code_prompt_hide: "Hide the code for plotting the figure"
+
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before reprojection")
+rast.plot(ax=ax[0])
```

Comment:
Detail, but here I would use `cmap="gray", add_cbar=False` to display in B&W.

Reply:
Done